src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java

```diff
@@ -82,28 +101,60 @@ public void sendEmailtoFollowers() {
               cal.add(Calendar.DAY_OF_MONTH, -1);
               lastCronJobExecDate = cal.getTime();
             }
-            String dailyRecommendationsIds =
-                technologyRecommendationDao
-                    .findAllRecommendationsIdsStartingFrom(lastCronJobExecDate);
-            String dailyCommentsIds =
-                technologyCommentDao.findAllCommentsIdsStartingFrom(lastCronJobExecDate);
-            List<Ref<TechGalleryUser>> followers = technologyFollowers.getFollowers();
-            if (!dailyRecommendationsIds.isEmpty() || !dailyCommentsIds.isEmpty()) {
-              for (Ref<TechGalleryUser> ref : followers) {
-                TechGalleryUser follower = ref.get();
-                emailService.push(follower, technology, dailyRecommendationsIds, dailyCommentsIds);
+
+            // Find new Activities in a technology
+            List<TechnologyRecommendation> dailyRecommendations =
```

Comment:
@brunoliberal Also move this to a method called something like 'findActivitiesByTechnology' or 'findDailyRecommendationsByTechnology'